enginetest/script_queries.go

```diff
@@ -33,12 +33,16 @@ type ScriptTest struct {
 	Expected []sql.Row
 	// For tests that make a single assertion, ExpectedErr can be set for the expected error
 	ExpectedErr *errors.Kind
+	// For tests that need to be skipped
+	Skip bool
```

Comment:
Don't do this, put tests that need to be skipped into their own blocks and call them from a different method (TestBrokenLoadFile)